hypothesis-python/docs/details.rst

```diff
@@ -591,6 +591,22 @@ fill in an argument from its type annotation.
     def test(a):
         pass
 
+
+``@given(...)`` can also be specified to fill all arguments from their type annotations.
+
+.. code:: python
+
+    @given(...)
+    def test(a: int, b: str):
+        pass
+
+
+    # is equivalent to
+    @given(a=integers(), b=text())
+    def test(a, b):
+        pass
```

Comment:
Let's expand it out to the intermediate step `@given(a=..., b=...)`.

Strictly speaking this is resolved to `a=from_type(int), b=from_type(str)`, which then *at execution time* goes to the standard strategies - unless someone has re-registered the strategies for those types!  Super pedantic, but that's kinda what we need here...